buildenv/jenkins/README.md

```diff
@@ -80,6 +105,8 @@ You can also request a Pull Request build from the Eclipse OpenJ9 repository - [
 - Ex. If you have a dependent change and only want one platform, depends comes last
     - `Jenkins test sanity zlinux jdk8 depends eclipse/omr#123`
 
+###### Note: When specifying a dependent change in an OpenJDK extensions repo, you can only build the SDK version that matches the repo where the dependent change lives. Eg. You cannot build JDK8 with a PR in openj9-openjdk-jdk9.
```

Comment:
Also here, remove the ref to the obsolete jdk9

Reply:
Done